test/configs/certs/rdns/ca.key

```diff
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
```

Comment:
Gist on how to generate these certs here: https://gist.github.com/wallyqs/18b241cf3eb7ae785dd09a9fbd40301c